sdks/python/apache_beam/dataframe/frames.py

```diff
@@ -200,6 +200,15 @@ def update(self, other):
 
   view = frame_base.wont_implement_method('memory sharing semantics')
 
+  @property
+  def str(self):
+    expr = expressions.ComputedExpression(
+        'str',
+        lambda df: df.str, [self._expr],
+        requires_partition_by=partitionings.Nothing(),
+        preserves_partition_by=partitionings.Nothing())
```

Comment:
Doesn't it preserve all partitionings?

Reply:
Doh yes this is basically an elementwise_method. Good catch